src/osd/osd_types.cc

```diff
@@ -1500,7 +1509,7 @@ void pg_pool_t::encode(bufferlist& bl, uint64_t features) const
     return;
   }
 
-  uint8_t v = 25;
+  uint8_t v = 26;
```

Comment:
Need double-check on this logic since I don't understand what the version override on line 1521 is doing.